src/renderer/helpers/assetHelper.ts

```diff
@@ -543,6 +543,12 @@ export const max1e10BaseAmount = (amount: BaseAmount): BaseAmount =>
  */
 export const to1e8BaseAmount = (amount: BaseAmount): BaseAmount => convertBaseAmountDecimal(amount, THORCHAIN_DECIMAL)
 
+/**
+ * Helper to convert a `BaseAmount`
+ * into `1e8` decimal based `BaseAmount`
```

Comment:
should update the comment as well into `1e10`?

Reply:
oh yes. will do